src/state.py

```diff
@@ -58,12 +65,12 @@ def is_end(self):
 
     def get_option_count(self):
         """Return the nuber of option in this state"""
-        pass
+        return self.options.count()
```

Comment:
do we really need this?

Reply:
yes, unless we replace `get_option` and `get_option_count` by a method `get_options` that return the list of options.

Reply:
get_options would be nice.